metadata.json

```diff
@@ -3,7 +3,7 @@
   "name": "Catppuccin",
   "description": "Soothing pastel theme for KiCad",
   "description_full": "4 Catppuccin flavors for KiCad, for both eeschema and pcbnew.",
-  "identifier": "com.github.waterlilly-lilly.catppuccin-kicad",
+  "identifier": "com.github.catppuccin.kicad",
```

Comment:
Is this fine to do? Reminds me of Java and wonder if this will end up breaking something.

Reply:
I believe it should be fine, it just might install a second package instead of updating the existing one for those who have it installed with the previous identifier which, to my knowledge, is just me.

Reply:
Sounds good, thanks for the explanation!